### ISSUE TITLE
Create a Python package to use translation models

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-training#802]
The Package should be able to either download or use already downloaded translation models and run translation efficiently. It means it should be able to utilize multiple CPUs.

The main challenge is compiling Marian. It should be possible to use prebuilt wheels, so that you don’t need to compile it locally as many Python scientific libraries do.

Another question to explore is whether we should include [bergamot-tranlsator ](https://github.com/browsermt/bergamot-translator/tree/main) at all or just run regular Marian or the [Bergamot fork of Marian](https://github.com/browsermt/marian-dev).

Some prior art: 
https://github.com/mozilla/translation-service
https://github.com/browsermt/bergamot-translator/tree/main/bindings/python

**Comment by marco-c** [comment]
I wonder if we can somehow upload the models on HuggingFace too. I guess we'd need to write a converter for the models like the one for opus-mt models.

**Comment by eu9ene** [comment]
Yeah, I also thought about this. Filed https://github.com/mozilla/firefox-translations-training/issues/804

**Comment by eu9ene** [comment]
If we convert to Pytorch and HF, it will be runnable with HF transformers but it will be slower than Marian.
